lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -270,9 +287,10 @@ sub request {
 
     push @$request_data, ('message_id' => $msg_id);
 
-    my $sent_future = $self->_send_request($request_data);
+    my $rpc_category_timeout = $self->_rpc_category_timeout($req_group);
```

Comment:
you can also ignore `rpc` in the names since they are obvious without that

```suggestion
    my $category_timeout = $self->_rpc_category_timeout($req_group);
```